docs/post-cloner.md

```diff
@@ -0,0 +1,71 @@
+# Post Cloner
+
+This feature adds a button to the post edit screen that allows a user to clone a post with its metadata and terms to a new, identical post.
+
+It is enabled by default but can be disabled via the configuration file:
+
+```json
+{
+	"extra": {
+		"altis": {
+			"modules": {
+				"cms": {
+					"cloner": false
+				}
+			}
+		}
+	}
+}
+```
+
+## Actions
+
+**`post_cloner_after_meta_data_copy: (int) $post_id`**
+
+Fires when post meta has been copied to cloned post.
+
+## Filters
+
+**`post_cloner_cloned_parent: (int) $parent Parent - defaults to original post ID.`**
+
+Allows overriding the parent that the new post should have.
+
+**`post_cloner_cloned_status: (string) $status Status - defaults to draft.`**
+
+Allows overriding the status that the new post should have.
+
+**`post_cloner_name_append: (string) $suffix String to append to the end of a cloned post's name. - defaults to '-cloned'`**
```

Comment:
I think these get a bit lengthy - rather than have a description of the parameter inline I was thinking just the parameter name and the default value if there is one. The explanation of the first parameter is effectively covered in the filter description below.